command.go

```diff
@@ -282,6 +297,17 @@ func copyFlagSetWithout(fs *pflag.FlagSet, without string) *pflag.FlagSet {
 	return fs2
 }
 
+func (inv *Invocation) curWords() (prev string, cur string) {
```

Comment:
If the CurWord can always be computed from AllArgs, why not just export this function and remove that variable? The more we can minimize state the better.

Reply:
Yeah, looks like this works - thanks!